README.md

```diff
@@ -580,6 +580,12 @@ Command | Description
 - `RUSTUP_UPDATE_ROOT` (default `https://static.rust-lang.org/rustup`)
   Sets the root URL for downloading self-updates.
 
+- `RUSTUP_USE_UNSAFE_SSL` (default: none)
+  If set, rustup will not validate the SSL certificate when downloading
+  files. This parameter should be used only in exceptional circumstances
+  when youre computer is behind a corporate proxy that injects its own
+  certificates into HTTPS connections.
+
```

Comment:
Should we ask for a very particular value, rather than merely "is set" since this is such a huge security issue?  Perhaps require it to be set to "accept-risks" or somesuch?